flink-cdc-common/src/main/java/com/ververica/cdc/common/schema/Schema.java

```diff
@@ -150,6 +151,14 @@ public static Schema.Builder newBuilder() {
         return new Builder();
     }
 
+    @Override
+    public String toString() {
+        final List<Object> components = new ArrayList<>(columns);
```

Comment:
We should show de pk information too, maybe you can refer flink's TableSchema related methods